README.md

```diff
@@ -161,6 +161,16 @@ Mixpanel.set({"$email": "elvis@email.com"});
 // Set People Properties Once (warning: if no mixpanel profile has been assigned to the current user when this method is called, it will automatically create a new mixpanel profile and the user will no longer be anonymous in Mixpanel)
 Mixpanel.setOnce({"$email": "elvis@email.com", "Created": new Date().toISOString()});
 
+// Add a new Group for this user.
+// @param Group key
+// @param A valid Mixpanel property type
+Mixpanel.setGroup('company', 'mixpanel');
```

Comment:
```suggestion
Mixpanel.addGroup('company', 'mixpanel');
```

Reply:
@davodesign84 why wasn't this change added? The README is currently incorrect